apps/test-tsx-app/admin.test.ts

```diff
@@ -0,0 +1,50 @@
+import test, { describe } from 'node:test';
+import { getPriciSdk, updateAdminUi } from './utils/prici-runner.js';
+import assert from 'node:assert';
+
+const authToken =
+  'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1lIjoiSm9obiBEb2UiLCJpYXQiOjE1MTYyMzkwMjJ9.e4vybEZSMruiGm8XgTxlD2QDGuOBz5l3gRfmjlmb1PY';
+
+describe('admin api route', async () => {
+  test('should throw 401 exception when ADMIN_UI is invalid', async () => {
+    const { quit, priciUBaseUrl } = await getPriciSdk();
+    const response = await fetch(`${priciUBaseUrl}/api/admin`);
+    await quit();
```

Comment:
The "quite()" should be inside an "afterEach" or "after all" callbacks.

If not, when the tests fail - they sometimes won't close the running instances.

Reply:
I separated the tests into 2 suites (one when ADMIN_UI is true and one when false) since I couldn't use quit() in "after" if I initialize our sdk inside the test itself.